src/services/mongodb/index.ts

```diff
@@ -62,13 +63,8 @@ const path = `mongodb://${user}:${password}@${address}/${db}`
 
 export const setupMongoConnection = async (syncIndexes = false) => {
   try {
-    await mongoose.connect(path, {
-      autoIndex: false,
-      useNewUrlParser: true,
-      useUnifiedTopology: true,
-      useCreateIndex: true,
-      useFindAndModify: false,
-    })
+    await mongoose.connect(path, { autoIndex: false })
+    mongoose.set("bufferCommands", false)
```

Comment:
why do you think that is needed? wouldn't retry be useful?

Reply:
oh ok. I don't remember why I put that here. I may have run into some issues at the beginning of working on this PR and set this settings up based on issues I may have had and problem some recommendation on a post in stackoverflow or something along this line.

 this is not specifically an intended long term settings I wanted to set

Reply:
Removed here https://github.com/GaloyMoney/galoy/pull/1406/commits/e77103c77d9694e935f3d0246a4bf12a22ce4ea5